src/components/PendingTxsSnackbar/store.test.js

```diff
@@ -0,0 +1,317 @@
+import { TransactionType, TransactionStatus } from 'constants';
+import { observable } from 'mobx';
+import PendingTxsSnackbarStore, { INIT_VALUES } from './store';
+import { mockResetTransactionList, mockAddTransaction } from '../../network/graphql/queries/';
+
+jest.mock('../../network/graphql/queries'); // block and manually mock our backend
+
+describe('PendingTxsSnackbarStore', () => {
+  let store;
+  const app = {
+    global: observable({ syncBlockNum: 0, syncPercent: 10 }),
+  }; // mock the appstore
+  const sleep = ms => new Promise(res => setTimeout(res, ms));
+
+  describe('Init', () => {
+    beforeEach(() => {
+      store = new PendingTxsSnackbarStore(app); // create a new instance before each test case
+      mockResetTransactionList();
+    });
+
+    it('Init with null tx list', async () => {
+      await store.init();
```

Comment:
move this to the root level `beforeEach()` since every test should run init() I believe.

Reply:
remove, already called in beforeEach